config/env/reporter_test.go

```diff
@@ -96,6 +96,27 @@ var _ = Describe("Reporter", func() {
 			})
 		})
 
+		Context("Set", func() {
+			It("sets the key with the value", func() {
+				Expect(syscall.Unsetenv("TIDEPOOL_TEST_JULIETTE")).To(Succeed())
+				reporter.Set("JULIETTE", "romeo")
```

Comment:
so i re-read this a few times and then remember it is because we are namespacing. Should that be evident in the code with a comment?

Reply:
Yea, I'll add documentation to the reporter itself.